examples/azure.json

```diff
@@ -27,5 +27,7 @@
     "useInstanceMetadata": true,
     "loadBalancerSku": "<loadbalancer-sku>",
     "excludeMasterFromStandardLB": false,
-    "maximumLoadBalancerRuleCount": 250
+    "maximumLoadBalancerRuleCount": 250,
+    "enableMultipleStandardLoadBalancers": false,
+    "tags": "a=b,c = d"
```

Comment:
let's remove spaces in example

Reply:
I just want to show that both formats are good.